### PR TITLE
HOTT-4810: Post migration cleanup

### PR DESCRIPTION
**Pull request by willfish**
### Jira link

https://transformuk.atlassian.net/browse/HOTT-4809
https://transformuk.atlassian.net/browse/HOTT-4810

### What?

I have added/removed/altered:

- [x] Fixed broken redirecting to govuk site in the production environment
- [x] Removed references to the VCAP_APPLICATION env var
- [x] Removed tf variable that is no longer consumed
- [x] Removed dead root route which is overridden by another route

### Why?

I am doing this because:

- These are just some things that have been missed

### CODE REVIEW

File: terraform/main.tf
Comment (amberstarlight) [inline terraform/main.tf:1]: Do we still need `GOVUK_APP_DOMAIN` in here? L65-66

Comment (willfish) [reply terraform/main.tf:1]: Sadly we don't but a gem we use that uses another gem marks it as mandatory

File: terraform/variables.tf [hunk 1]
```diff
@@ -45,8 +45,3 @@ variable "memory" {
   description = "Memory to allocate in MB. Powers of 2 only."
   type        = number
 }
-
-variable "beta_search_enabled" {
```
Comment (amberstarlight) [inline terraform/variables.tf:49]: We can get rid of the var in the corresponding `config_<environment>.tfvars` files now, too

**Review by amberstarlight — CHANGES_REQUESTED** [review]

File: terraform/variables.tf (hunk 1 above)
Comment (willfish) [reply terraform/variables.tf:49]: Done

**Review by amberstarlight — APPROVED** [review]

**Pull request merged**